src/bio/bio_buffer.c

```diff
@@ -1032,44 +1107,86 @@ int
 bio_readv(struct bio_io_context *ioctxt, struct bio_sglist *bsgl,
 	  d_sg_list_t *sgl)
 {
-	int	rc;
+	struct media_error_msg		 mem = { 0 };
```

Comment:
Because we don't wait for media error message completion, this should be dynamically allocated on error. (in rw_completion()).

Reply:
done